dot/state/block_test.go

```diff
@@ -461,12 +462,7 @@ func TestFinalization_DeleteBlock(t *testing.T) {
 	after := bs.bt.GetAllBlocks()
 
 	isIn := func(arr []common.Hash, b common.Hash) bool {
```

Comment:
We can get rid of `isIn()` completely and just call `slices.Contains()` where it was used:
```suggestion
	// assert that every block except finalised has been deleted
	for _, b := range before {
		if b == fin {
			continue
		}
		if slices.Contains(after, b) {
			continue
		}
```

Reply:
Good idea!

Modified. Please review again.